CMakeLists.txt

```diff
@@ -161,3 +161,6 @@ endif()
 
 # Install build system files
 install(DIRECTORY cmake DESTINATION share/opm)
+
+# Install tab completion skeleton
+install(FILES etc/opm_bash_completion.sh.in DESTINATION share/opm/etc)
```

Comment:
it's the template file so i figured we'd keep it in opm directories.

Reply:
I meant using `${CMAKE_INSTALL_DATAROOTDIR}/opm/etc` as the installation. But that would mean that the scripts using it would need to configurable. That might be overkill. Leave it as is